module-example/prisma/extesion.ts

```diff
@@ -0,0 +1,17 @@
+import { Prisma } from '@prisma/client';
+
+export const prismaExtension = Prisma.defineExtension((client: any) => client.$extends({
```

Comment:
I think prisma client is this magical object that is pre-built by the `prisma generate` command based on the prisma schema. So to avoid `any` and use correct version of the client, you need to import a different prisma client. But maybe this introduces unnecessary interdependency, which we can sacrifice over type safety. I have to try/think about it

Reply:
I would leave it for now to not introduce interdependency